etc/profile-m-z/transmission-qt.profile

```diff
@@ -15,6 +15,9 @@ private-bin transmission-qt
 #dbus-user.own com.transmissionbt.Transmission.*
 #dbus-user.talk org.freedesktop.Notifications
 
+# If you need system tray support, add this line to your transmission-qt.local
```

Comment:
I would just enabled it with the condition.

Reply:
Will fix, thanks.